tests/testthat/test-lifecycle_admiral.R

```diff
@@ -0,0 +1,41 @@
+## Test 1:  Message is sent to user ----
+test_that("lifecycle_admiral Test 1:  Message is sent to user", {
+  example_fun <- function(dataset) {
+    deprecate_inform("1.0.0", "example_fun()", "example_fun2()")
+    assert_data_frame(dataset, required_vars = exprs(STUDYID, USUBJID))
+  }
+
+  data <- dplyr::tribble(
+    ~STUDYID, ~USUBJID,
+    "xyz",    123,
+    "xyz",    456
+  )
+
+  expect_snapshot(
+    example_fun(data),
+  )
```

Comment:
We could follow this with an `expect_silent(example_fun(data))`, since it should only print the first time.

Reply:
I think there is some issue with the testthat environments that is causing an issue here.  see anything amiss? 

Code doesn't create the message in the second call, but fails in the testing environment.

![image](https://github.com/user-attachments/assets/0549f74c-0270-42e0-a74d-a7c5d14ce26c)

Reply:
Hmmm, tricky! Does this work? We should only get the message from the first, and they are both running in the same env (which hopefully, solves the failure).

```r
expect_snapshot({
  example_fun(data)
  example_fun(data)
})
```

Reply:
Interesting it duplicates the message in the snapshot.

But when I call it in my environment is doesn't duplicate the message!!  

A Mystery

Reply:
🤷🏼 we can just move on then hahaha

Reply:
I asked the AI gods and it gave me some strange suggestions which I think were kind of weak.

oh well...maybe as we place these in admiral functions we will get some other weirdness surfaced!